wom/src/main/scala-2.12/wdl4s/wom/executable/Executable.scala

```diff
@@ -0,0 +1,78 @@
+package wdl4s.wom.executable
+
+import cats.syntax.validated._
+import lenthall.Checked
+import lenthall.validation.ErrorOr._
+import shapeless.Coproduct
+import wdl4s.wdl.types.WdlType
+import wdl4s.wdl.values.WdlValue
+import wdl4s.wom.callable.Callable
+import wdl4s.wom.executable.Executable.{DelayedCoercionFunction, InputParsingFunction, ResolvedExecutableInputs}
+import wdl4s.wom.graph.Graph.ResolvedExecutableInput
+import wdl4s.wom.graph.GraphNodePort.OutputPort
+import wdl4s.wom.graph._
+
+object Executable {
```

Comment:
Is there any way to duplicate less of this file between 2.11 and 2.12? A transitive import perhaps?

Reply:
I'll give it a shot